lib/basedriver/commands/images.js

```diff
@@ -45,17 +60,37 @@ commands.compareImages = async function compareImages (mode, firstImage, secondI
     case MATCH_TEMPLATE_MODE.toLowerCase():
       // firstImage/img1 is the full image and secondImage/img2 is the partial one
       result = await imageUtil.getImageOccurrence(img1, img2, options);
+
+      if (options.multiple) {
+        for (const element of result.multiple) {
+          convertVisualizationToBase64(element);
+        }
+
+        return result.multiple;
+      }
       break;
     default:
       throw new errors.InvalidArgumentError(`'${mode}' images comparison mode is unknown. ` +
         `Only ${JSON.stringify([MATCH_FEATURES_MODE, GET_SIMILARITY_MODE, MATCH_TEMPLATE_MODE])} modes are supported.`);
   }
-  if (!_.isEmpty(result.visualization)) {
-    result.visualization = result.visualization.toString('base64');
-  }
+
+  convertVisualizationToBase64(result);
```

Comment:
we could return the converted result from this method: `return convertVisualizationToBase64(result);`

Reply:
How does this look now?

Reply:
perfect